scarb/src/bin/scarb/args.rs

```diff
@@ -241,6 +241,10 @@ pub struct InitArgs {
     /// Do not initialize a new Git repository.
     #[arg(long)]
     pub no_vcs: bool,
+
+    /// Initialize snforge setup.
```

Comment:
```suggestion
    /// Use a Starknet Foundry Forge template.
```

<img width="1082" alt="image" src="https://github.com/software-mansion/scarb/assets/3450050/01ceba73-c103-4bce-ab1e-9619483d6680">